### DISCUSSION TITLE
Social Media and Viseron?

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#359] [Q&A] [answered]
Do you have Viserion on social media? FB? IG? others?

Thanks

**Comment by roflcoopter** [comment, marked as answer]
Sadly no, I'm thinking of making a discord server or something when v2 is released, for support, questions, announcements etc.

Gonna try to post more on Reddit and stuff to see if I can get some more traction

**Reply by gniknalu** [reply]
> Sadly no, I'm thinking of making a discord server or something when v2 is released, for support, questions, announcements etc.
> 
> Gonna try to post more on Reddit and stuff to see if I can get some more traction

I'm more than happy to assist in any of your endeavors. If you decide to do a FB group/service - it doesn't have to be elaborate but just a place to put updates/advertise.  I'll be glad to assist there as well. Thanks

**Comment by testerz** [comment]
any updates about discord ?
I think having a subreddit would be beneficial so that people can share articles and guides and experinces and ask questions and actually get answered from people who use it, not a lot of members will be keen to check github for questions etc ...

**Reply by roflcoopter** [reply]
I agree, sorry for not going through with this.
I actually started a Discord a while back but got stuck on how to best setup rules, bots, channels etc. Will take a look at it again

**Reply by roflcoopter** [reply]
Created a server and did my best to set it up, but not sure how well i did 😄

Will announce it wide if it looks good, could you join and give some feedback on the channels etc?

https://discord.gg/MxP5xpyvk7

**Reply by roflcoopter** [reply]
@john- are you interested in checking out the discord server (if you have discord ofc)?

Want some extra eyes on it before announcing it, and you've been participating in quite a lof of threads here on Github

**Reply by roflcoopter** [reply]
And of course @gniknalu, are you still using Viseron? Would be happy to see you over at the Discord as well if thats the case

**Reply by ulanking** [reply]
> And of course @gniknalu, are you still using Viseron? Would be happy to see you over at the Discord as well if thats the case

Still using Viseron - just a little late to my own party - LOL

**Reply by roflcoopter** [reply]
Yeah only took me 4 years to set up said party 🙃

**Reply by testerz** [reply]
discord looking good so far, hope many users actually end up using which would be helpfull to all.
from what i noticed many homelabers/selfhosters are on reddit as they can search for topic and can reach exact post/reply with link, no app needed and its simply mostly text, but discord for now is a good first step there are still many big communities using it.
